src/gsi/read_fl_hdob.f90

```diff
@@ -533,6 +533,9 @@ subroutine read_fl_hdob(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,si
                write(6,*) 'READ_FL_HDOB: bad lat/lon values: ', obsloc(1,1),obsloc(2,1)              
                cycle loop_readsb2     
            endif
+!          Convert degree West to degree East for the HDOBs data location for
+!          the global model
```

Comment:
Please remove this comment given past confusion over whether or not to remap negative longitude to positive.   We perform this remapping whether running in global or regional mode.   Line 536 can read

"Convert degree West to degree East for the HDOBs data location".

Remove line 537.

Reply:
Removed.